src/META-INF/plugin.xml

```diff
@@ -57,12 +57,18 @@
     <internalFileTemplate name="Go File"/>
 
     <!-- project -->
+    <projectConfigurable instance="com.goide.ui.GoConfigurable" groupId="language"/>
```

Comment:
Please delete configurable definition form plugin.xml and add its constructing to GoConfigurableProvider, I'll clean up this later.